src/scss/attendee/theme.scss

```diff
@@ -26,6 +26,8 @@
   --overlay-fg: rgb(255, 255, 255);
   --overlay-bg: rgba(99, 42, 199, 0.7);
 
+  --mask-bg: rgb(0, 0, 0, 0.5);
```

Comment:
This is not going to work, `rgba` is the one that handles `alpha`

Reply:
[1ec7d1e](https://github.com/sparkletown/sparkle/pull/2973/commits/1ec7d1e6854fb0c34f6633c95e8d4ec95fe9603c)